docs/getting_started.md

```diff
@@ -0,0 +1,89 @@
+# Getting Started
+
+## What is the goal of Inago?
+
+Inago is a deployment tool that manages groups of unit files to deploy them to
+a fleet cluster similar to `fleetctl`. Since `fleetctl` is quite limited, Inago
+aims to abstract units away and provide more sugar on top like update
+strategies. That way the user can manage unit files more easily.
+
+## Prerequisites
+
+Inago requires a certain directory structure and unit file names to make the
+tool actually work. There must be a group folder with unit files in it. For details see the [Unit File Structure](structure.md) chapter.
+
+## Working with Inago
+
+The basic commands you can use with Inago are `submit`, `start`, `stop`, `destroy`, and `status`. The subject of your commands is always a unit group as defined by a group folder.
+
+### Submit
+
+Submitting groups enables deploying multiple slices using one command.
+
+```
+inagoctl submit myapp 3
+```
+
+This assumes there is a directory `myapp` and that the unit files in that directory cary an `@` and thus are to be sliced. The argument `3` tells Inago to create 3 slices of the units found in the group directoy. Inago looks up the
+directory `myapp`. So lets imagine there are two unit files within
+`myapp`.
+
+```
+myapp_some_unit_name@.service
+myapp_some_other_unit_name@.service
+```
+
+Given the scaling argument `3` and the unit file names the resulting unit file
+names that are going to be created for our `submit` command should look
+something like the following.
+
+```shell
+# first slice
+myapp_some_unit_name@s8k.service
+myapp_some_other_unit_name@s8k.service
+
+# second slice
+myapp_some_unit_name@0ds.service
+myapp_some_other_unit_name@0ds.service
+
+# third slice
+myapp_some_unit_name@h38.service
+myapp_some_other_unit_name@h38.service
+```
+
+### Start, Stop, Destroy
+
+Once you have submitted a group like explained above, you can then use Inago to start, stop, or destroy that group with a single command each.
+
+```
+inagoctl start myapp
+
+inagoctl stop myapp
+
+inagoctl destroy myapp
+```
+
+### Status
+
+Using the `status` command you can enquire the current status of your group and compare desired and actual states of each slice. By default the substates of the units of each group slice are aggregated as long as they are consistent accross the slice.
+
+```
```

Comment:
IIRC if use nohighlight if you are not sure as using no tag could result in unintended highlighting in the docs page